Algorithms.Tests/Financial/PresentValueTests.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using Algorithms.Financial;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Algorithms.Tests.Financial;
+
+public static class PresentValueTests
+{
+    [Test]
+    public static void Present_Value_General_Tests()
```

Comment:
Please use TestCase to create parameterized tests

Reply:
done